frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -374,35 +374,35 @@ end
 function BookInfo:getCoverImage(document, file, force_orig)
     local curr_file = document and document.file
     local cover_bb
-    -- check for a custom cover (orig cover is forcibly requested in "Book information" only)
+
     if not force_orig then
         local custom_cover = DocSettings:findCustomCoverFile(file or curr_file)
         if custom_cover then
             local cover_doc = DocumentRegistry:openDocument(custom_cover)
             if cover_doc then
                 cover_bb = cover_doc:getCoverPageImage()
                 cover_doc:close()
-                return cover_bb, custom_cover
+                if cover_bb then return cover_bb end -- Early return
```

Comment:
What's the point of this change?
`cover_doc` is an opened graphic file, it **must** provide `cover_bb`.
And we **do have** an early return.

Besides, you've missed the second return value:
https://github.com/koreader/koreader/blob/0386e2619eb5c11367980741af062d3865ea4e8c/plugins/coverimage.koplugin/main.lua#L114